lib/server.js

```diff
@@ -331,6 +331,11 @@ class Server extends EventEmitter {
 
         // Command line
         var commandLine = f('%s %s', self.binary, commandOptions.join(' '));
+
+        if (self.logger.isInfo()) {
```

Comment:
why was this moved before? Just to match other files?

Reply:
Yeah, just to match `mongos`. Small fix that I figured would be fine.